src/Connections/GitlabRepo.ts

```diff
@@ -239,13 +239,13 @@ export class GitLabRepoConnection extends CommandConnection {
         let content = `**${event.user_name}** pushed [${event.total_commits_count} commit${event.total_commits_count > 1 ? "s": ""}](${commitsurl})`
         + ` to [\`${branchname}\`](${branchurl}) for ${event.project.path_with_namespace}`;
 
-        if (event.commits.length >= 2) {
+        if (displayedCommits >= 2) {
             content += `\n - ${commits}\n`;
-        } else if (event.commits.length === 1) {
-            content += ` - ${commits}`;
+        } else if (displayedCommits === 1) {
+            content += `: ${commits}`;
         }
         if (tooManyCommits) {
-            content += `. View [${event.total_commits_count - 1} more](${commitsurl}).`;
+            content += `, and [${event.total_commits_count - 1} more](${commitsurl}) commits`;
         }
```

Comment:
This behaviour could be moved inside `else if (displayedCommits === 1)` as it is only triggered then.
What do you think, readability-wise?